tests/monster_test.fbs

```diff
@@ -6,6 +6,11 @@ namespace MyGame.Example2;
 
 table Monster {}  // Test having same name as below, but in different namespace.
 
+/// This will test a potential "ambiguous call to overloaded function" using the same data from different namespace.
+table MonsterStatus {
```

Comment:
Makes sense, I just wanted you to have the option to commit this if you wanted since there isn't a test case in covering this issue.